game/lib/drag-drop.js

```diff
@@ -0,0 +1,86 @@
+dragula([
```

Comment:
Wait a minute... this is not your code. Why aren't you importing it from npm and using browserify to bundle it?